### ISSUE TITLE
RenameVariables should have option to skip normalizing ${_}

### ISSUE THREAD

**Issue by bhirsz** [MarketSquare/robotframework-tidy#540]
There should be a way to define such variable without normalizing it to ${}:

```
${_}  ${second_return}    Keyword Call  # I dont need first return value so I assigned it to ${_}
```

**Comment by bhirsz** [comment]
Same goes for some special variables (like ${/n}, it looks better than ${/N}). I'm also on fence for ${True} -> ${TRUE}. I could introduce some variables that should generally be ignored (it could be configurable). If you have some examples where you think the RenameVariables shouldn't rename - please let me know.

**Comment by bhirsz** [comment]
Fixed - it will be released in 4.4.0 today.
